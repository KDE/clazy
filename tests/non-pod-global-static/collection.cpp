#include <QtCore/QList>
#include <QtCore/QString>

const int myArr[] = {1,2,3}; // OK
const QString strArr[] = { // Warn, once
  QString("testme"),
  QString("testme2")
};
static QList<QString> s_globalStuff{ // Warn, once
  QString("Hello"),
  QString("fellow"),
  QString("KDE"),
  QString("dev")
};
