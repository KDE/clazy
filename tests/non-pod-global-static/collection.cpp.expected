non-pod-global-static/collection.cpp:5:1: warning: non-POD static (QString strArr) [-Wclazy-non-pod-global-static]
non-pod-global-static/collection.cpp:9:1: warning: non-POD static (QList s_globalStuff) [-Wclazy-non-pod-global-static]
